tooling/ast_fuzzer/src/program/tests.rs

```diff
@@ -86,9 +86,9 @@ fn test_modulo_of_negative_literals_in_range() {
     assert_ssa_snapshot!(ssa, @r"
     acir(inline) fn main f0 {
       b0():
-        jmp b1(i64 18446744073709551612)
+        jmp b1(i64 -4)
       b1(v0: i64):
-        v3 = lt v0, i64 18446744073709551615
+        v3 = lt v0, i64 -1
```

Comment:
I remember we discussed that multiple Field values can be mapped to e.g. `i8` value of `0`. We don't have any ambiguity when it comes to negatives, right?

Reply:
I think no. At least in the range of the bit_size (`0..2^bit_size - 1`) there's no ambiguity.

It could happen that the field value falls outside of that range. In that case I decided to format the number as-is instead of erroring because otherwise a test called `does_not_crash_on_truncated_division_with_large_denominators` fails, but I don't understand why that test exists or how it could happen (in my mind we should never produce such code).